### ISSUE TITLE
[Feat] [Bug] Stop scrolling and zooming out when clicking icons; etc.

### ISSUE THREAD

**Issue by SerggioC** [vostpt/greve-transportes-perigosos#138]
When we are zoomed in in the map and click an icon we are zoomed out automagically, which is annoying.

When clicking an icon the popup info window is huge!
Can you make this smaller and add clear information about available fuels? Pricing would be nice too.

In mobile devices, which the majority of accesses come from I believe, the site has some issues.
When scrolling, the whole page scrolls instead of the map.
The legend gets behind the search field...

**Comment by JorgeMiguelGomes** [comment]
Hi @SerggioC thank you for the input.
It's totally impossible to get more information, than the one we are already providing, about availability let alone pricing. We've developed, and made available, an API for brands to use, where they can inject that information into the platform. 

Regarding mobile usability:
- Scrolling: Can you please tell us what your devices's screen resolution is? **We can't replicate that issue** 
- Zooming out on click: **We'll look into this**
@Cotemero regarding this issue we will address, as a low priority task, the zooming out issue.

**Comment by SerggioC** [comment]
Hi @JorgeMiguelGomes,

Check out this video recording on a Huawei P8 lite. you'll be able to see the issues. 
https://www.youtube.com/watch?v=RMQtOVUn0tQ
Device res: 1920*1080

The map starts scrolled way up with the page probably because of previous scroll in the selections page.
Scrolling on the toolbar also scrolls the entire page with the map.
The menu icon on the toolbar does nothing when on the map... 
When we click the icons the popup info window is huge and out of view. We could have smaller pump icons there and a clear description if available or not. 

ok you have margin for improvements 😃 

Thanks all for the tool.

**Comment by miguelantoniosantos** [comment]
Can you try to force refresh the page? or with a different browser on the same cellphone, this issue was fixed for many cellphones sometime ago, don't know if you are having cache issues

**Comment by SerggioC** [comment]
No, same thing happening after cache cleared on Chrome, Brave. and samsung internet browser on a tablet.

**Comment by miguelantoniosantos** [comment]
Can someone else recreate it?
